<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_welcomr">
      <widget class="QPushButton" name="pdb_start">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>390</y>
         <width>150</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bradley Hand ITC</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pdb_exit">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>390</y>
         <width>150</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bradley Hand ITC</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
      <widget class="QLabel" name="SOKOBAN">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>140</y>
         <width>451</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
         <pointsize>48</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>SOKOBAN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>270</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Choose a map</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>270</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Map 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map 11</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="page_game">
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>0</y>
         <width>3</width>
         <height>600</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>310</y>
         <width>200</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_guanqia">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>0</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Map 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_time">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>320</y>
         <width>161</width>
         <height>251</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Timer: </string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>50</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_step">
       <property name="geometry">
        <rect>
         <x>607</x>
         <y>74</y>
         <width>161</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Steps:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
